guide/README.md

```diff
@@ -1233,6 +1254,7 @@ block. In the example above, the default instrument is set to piano,
 so any note that is not inside of a *Set instrument* block will be
 played using the piano synthesizer. The first note in this example is
 piano; the second note is guitar; and the third is piano.
+[RUN LIVE](https://musicblocks.sugarlabs.org/index.html?id=1725869737763127&run=True)
```

Comment:
I haven't looked at all of these examples, but I did look at this one. The example you generated does not match the text or illustration. Please double check these live examples.

Reply:
Hello @walterbender,

I checked the "RUN LIVE" link and found that I am unable to use the "Set Default Instrument" block. Whenever I select it, the block is converted to an "Unknown" block.
![Screenshot 2024-09-10 224437](https://github.com/user-attachments/assets/aea3976c-9e12-4ca0-bf4a-9d5a196964c6)

Reply:
Also @walterbender , could you please advise on what to do about this "RUN LIVE" link? I am currently unable to use it.
https://musicblocks.sugarlabs.org/index.html?id=1725869737763127&run=True

Reply:
Oh. There seems to be a regression with the set default instrument block. We'll have to sort it out. Maybe leave that example out of your PR, since it is not correct? And maybe you can write up an issue on the broken block?

In general, your PR is very nice. But it is only by chance that I picked up on the errant example. In the future, it would be good to make a note of such discrepancies in your commit message or PR description.

Reply:
okay @walterbender  i will do that.

Reply:
Hello @walterbender , 
I have raised the regression issue, but I'm unable to solve it because I can't open the Music Blocks project on my localhost. I have tried everything but it just load, and it was working two days ago, but now it's not opening. Until this problem is resolved, I don't think I'll be able to contribute to any issues other than documentation.

https://github.com/user-attachments/assets/9300a3d7-e831-404d-8029-0ce70b2ef7c5

Could you please guide me on what to do to open the Music Blocks project on my localhost again?

Reply:
Yes, I have tried running it multiple times on my local server, and there are no errors in my console. I also tried the same setup on another laptop, and it worked, but I’m not sure why it's not working on mine.

Additionally, I have added some new links to the documentation and double-checked the other live links, so you can review it again and let me know if there are any issues with my PR this time. As for the regression issue, I have assigned it to @omsuneri.

Reply:
Try running by opening the file (index.html) directly from your browser. You should see errors in the console if it doesn't load properly. (The fact that it worked on another machine makes me think that perhaps the version you are running is not up-to-date with the master branch. What is the last commit on master you are seeing?)